packages/dev/docs/src/docs.css

```diff
@@ -501,19 +501,24 @@ h2.sectionHeader {
 .pageHeader {
   position: fixed;
   top: 0;
-  left: 0;
-  display: flex;
+  display: inline-flex;
   align-items: center;
   justify-content: space-between;
   padding: 0 8px;
   box-sizing: border-box;
   height: var(--spectrum-global-dimension-size-600);
-  width: 100%;
+  [dir='ltr'] & {
+    right: 0;
+  }
+  [dir='rtl'] & {
+    left: 0;
+  }
 }
 
 /* hamburger menu should be hidden so that it doesn't receive focus before the sidenav */
  .hamburgerButton {
    visibility: hidden;
+   display:none;
```

Comment:
so, visibility: hidden isn't needed anymore?

Reply:
yeah, it was easier to keep both through the merge, i can remove it now